### ISSUE TITLE
Failed to arm Action lacks trigger entity

### ISSUE THREAD

**Issue by maximoei** [nielsfaber/alarmo#597]
### Alarmo Version

v1.9.6

### HA Version

2022.10

### Bug description

When using the Failed to arm Action, the Tasks can no longer make use of the `trigger` object to create for example a notification with the reason why the alarm can not arm. Previously, task below would work:

`- service: notify.telegram_group
  data:
    message: |
      Arm failed:  {% for entity_id in trigger.event.data.sensors %}
        {{ state_attr(entity_id, 'friendly_name') }} is {{ states(entity_id) }}
      {% endfor %}
`

### Steps to reproduce

Create a Failed to arm Action, try to use the trigger object in a Task

### Relevant log output

```shell
Logger: homeassistant.helpers.template
Source: helpers/template.py:135
First occurred: October 23, 2022 at 8:39:45 AM (8 occurrences)
Last logged: 7:52:06 PM

Template variable error: 'trigger' is undefined when rendering 'Failed to arm: {% for entity_id in trigger.event.data.sensors %} {{ state_attr(entity_id, 'friendly_name') }} is {{ states(entity_id) }} {% endfor %}'
```

**Comment by nielsfaber** [comment]
I am assuming this is part of an automation outside of Alarmo?
What is used as trigger of this automation?

**Comment by maximoei** [comment]
No, this is inside the Action in Alarmo, using YAML mode to create a custom Task action

**Comment by nielsfaber** [comment]
I don't know how to help with this.
Alarmo doesn't contain any code which handles (jinja) templates, so this functionality must reside within HA instead.
Frankly I'm surprised this ever worked, as there is no trigger object used by or forwarded from Alarmo when it executes the notification service.

**Comment by maximoei** [comment]
Basically the objective is exactly the same as when doing the alarm trigger: sending a notification that explains why. Should I repost as a feature request?
